packages/devextreme/js/__internal/grids/grid_core/editor_factory/m_editor_factory.ts

```diff
@@ -61,7 +61,19 @@ export class EditorFactory extends ViewControllerWithMixin {
 
   _getFocusedElement($dataGridElement) {
     const rowSelector = this.option('focusedRowEnabled') ? 'tr[tabindex]:focus' : 'tr[tabindex]:not(.dx-data-row):focus';
-    const focusedElementSelector = `td[tabindex]:focus, ${rowSelector}, input:focus, textarea:focus, .dx-lookup-field:focus, .dx-checkbox:focus, .dx-switch:focus, .dx-dropdownbutton .dx-buttongroup:focus, .dx-adaptive-item-text:focus`;
+    const focusedElementSelector = `
+      td[tabindex]:focus,
+      ${rowSelector},
+      input:focus,
+      button:focus,
+      textarea:focus,
+      div[tabindex]:focus,
+      .dx-lookup-field:focus,
+      .dx-checkbox:focus,
+      .dx-switch:focus,
+      .dx-dropdownbutton
+      .dx-buttongroup:focus,
+      .dx-adaptive-item-text:focus`;
```

Comment:
Let's remove spaces and new line chars from this selector string.
Something like this:
```suggestion
    const focusedElementSelector = `
      td[tabindex]:focus,
      ${rowSelector},
      input:focus,
      button:focus,
      textarea:focus,
      div[tabindex]:focus,
      .dx-lookup-field:focus,
      .dx-checkbox:focus,
      .dx-switch:focus,
      .dx-dropdownbutton
      .dx-buttongroup:focus,
      .dx-adaptive-item-text:focus`.replace(/\n| /g, ''); // after the "|" symbol there is a space!
```

Because when you create a string like this, you add additional spaces and "new line" chars to this string
```typescript
const myStr = `
  a,
  b,
  c`;
  console.log(myStr); // "\n  a,\n  b,\n  c"
  console.log(myStr.replace(/ /g, '_')); // "\n__a,\n__b,\n__c"
  console.log(myStr.replace(/\n| /g, '')); // "a,b,c"
``` 

Maybe the jQuery clears a selector string of these symbols (I'm not 100% sure), but in my opinion, it's better to replace it here :)